scripts/tasks/storybook.ts

```diff
@@ -83,6 +83,7 @@ export function buildStorybookTask() {
       mode: 'static',
       outputDir: path.join(process.cwd(), 'dist/storybook'),
     };
+    console.log('Storybook options:', JSON.stringify(options, null, 2));
```

Comment:
do we need to spam users with this output ? I'd suggest to take similar approach like you did in webpack `if (process.env.TF_BUILD) {`

Reply:
I could do that, though it's less important since this output is small

Reply:
up to you then :)